include/modem/nrf_modem_lib.h

```diff
@@ -14,6 +14,9 @@
  * @brief API of the SMS nRF Modem library wrapper module.
  */
 
+#ifndef NRF_MODEM_LIB_H_
+#define NRF_MODEM_LIB_H_
```

Comment:
Shouldn't this be located before the doxygen group is opened (i.e. right after the copyright notice)?

Reply:
I see there is a mix whether this is located before or after the doxygen group, even in the `include/modem` folder. However, as the doxygen group is closed first, I will put it before.